admin/resources/js/components/pool/CreatePool.tsx

```diff
@@ -0,0 +1,219 @@
+import * as React from "react";
+import { FormProvider, SubmitHandler, useForm } from "react-hook-form";
+import { useIntl } from "react-intl";
+import {
+  Classification,
+  CmoAsset,
+  CreatePoolInput,
+  CreatePoolMutation,
+  OperationalRequirement,
+  Pool,
+  useCreatePoolMutation,
+  useGetCreatePoolDataQuery,
+  User,
+} from "../../api/generated";
+import { notEmpty } from "../../helpers/util";
+import errorMessages from "../form/errorMessages";
+import Input from "../form/Input";
+import MultiSelect from "../form/MultiSelect";
+import Select from "../form/Select";
+import Submit from "../form/Submit";
+import TextArea from "../form/TextArea";
+
+type Option<V> = { value: V; label: string };
+
+type FormValues = Pick<Pool, "name" | "description"> & {
+  assetCriteria: string[] | undefined;
+  classifications: string[] | undefined;
+  essentialCriteria: string[] | undefined;
+  operationalRequirements: string[] | undefined;
+  owner: string;
+};
+
+interface CreatePoolFormProps {
+  classifications: Classification[];
+  cmoAssets: CmoAsset[];
+  locale: "en" | "fr";
+  operationalRequirements: OperationalRequirement[];
+  users: User[];
+  handleCreatePool: (
+    data: CreatePoolInput,
+  ) => Promise<CreatePoolMutation["createPool"]>;
+}
+
+export const CreatePoolForm: React.FunctionComponent<CreatePoolFormProps> = ({
+  classifications,
+  cmoAssets,
+  locale,
+  operationalRequirements,
+  users,
+  handleCreatePool,
+}) => {
+  const intl = useIntl();
+  const methods = useForm<FormValues>();
+  const { handleSubmit } = methods;
+
+  const formValuesToSubmitData = (values: FormValues): CreatePoolInput => ({
+    ...values,
+    assetCriteria: {
+      sync: values.assetCriteria,
+    },
+    classifications: {
+      sync: values.classifications,
+    },
+    essentialCriteria: {
+      sync: values.essentialCriteria,
+    },
+    operationalRequirements: {
+      sync: values.operationalRequirements,
+    },
+    owner: { connect: values.owner },
+  });
+
+  const onSubmit: SubmitHandler<FormValues> = async (data: FormValues) => {
+    await handleCreatePool(formValuesToSubmitData(data))
+      .then(() => {
+        // TODO: Navigate to pool candidate dashboard.
```

Comment:
Is there another issue for this _TODO_?

Reply:
Yup https://github.com/GCTC-NTGC/gc-digital-talent/issues/195